src/pixelator/annotate/__init__.py

```diff
@@ -433,4 +437,12 @@ def anndata_metrics(adata: AnnData) -> AnnotateAnndataStatistics:
     if "doublet_size_threshold" in adata.uns:
         metrics["doublet_size_threshold"] = adata.uns["doublet_size_threshold"]
 
+    if "is_potential_doublet" in adata.obs:
+        metrics["fraction_potential_doublets"] = adata.obs[
```

Comment:
If there are no `is_potential_doublet` here, what will it happen when run mean?

Reply:
There will be a key error, that's why we check for it first.

Reply:
But when `adata.obs["is_potential_doublet"]` returns empty and we called `mean()` - what happens then?

Reply:
We only run the mean() function if "is_potential_doublet" exists as a column in adata.obs. Otherwise that part is not run and "fraction_potential_doublets" remains the default value which is 0 right now but following the discussion we're going to change it to None.